packages/webapp-events/package.json

```diff
@@ -1,22 +1,22 @@
 {
-  "description": "Events shared between Wire's web app and desktop app.",
-  "devDependencies": {
-    "typescript": "4.8.4"
-  },
-  "files": [
-    "src/**/*.js*",
-    "src/**/*.d.ts"
-  ],
-  "license": "GPL-3.0",
-  "main": "src/index",
-  "name": "@wireapp/webapp-events",
-  "repository": "https://github.com/wireapp/wire-web-packages/tree/main/packages/webapp-events",
-  "scripts": {
-    "build": "tsc",
-    "build:node": "tsc",
-    "clean": "rimraf \"src/*{.js,.js.map,.d.ts}\"",
-    "dist": "yarn clean && yarn build",
-    "test": "exit 0"
-  },
-  "version": "0.14.9"
+    "name": "@wireapp/webapp-events",
+    "license": "GPL-3.0",
+    "description": "Events shared between Wire's web app and desktop app.",
+    "repository": "https://github.com/wireapp/wire-web-packages/tree/main/packages/webapp-events",
+    "files": [
+        "lib"
+    ],
+    "main": "lib/index",
+    "devDependencies": {
+        "@types/rimraf": "^3.0.2",
+        "rimraf": "^3.0.2",
+        "typescript": "4.8.4"
+    },
+    "scripts": {
+        "build": "yarn clean && tsc",
+        "clean": "rimraf lib",
+        "dist": "yarn build",
+        "test": "exit 0"
+    },
+    "version": "0.14.9"
```

Comment:
I think you have indentation problem here

Reply:
Fixed it with my latest commit. package.json has been on the prettier ignore list, not anymore